### ISSUE TITLE
v2.16.0 or v2.16.1 Failed unlocking source: Invalid credentials content (unknown signature)

### ISSUE THREAD

**Issue by wodin** [buttercup/buttercup-browser-extension#345]
Since upgrading to v2.16.0 (or v2.16.1) on Firefox the extension is unable to unlock my archive that is stored on a WebDAV server. The problem first happened on Firefox:

<img width="505" alt="Firefox" src="https://user-images.githubusercontent.com/4579020/86316563-5c852000-bc2d-11ea-9ce4-acd2484a75f2.png">

I then switched to Chrome which still had v2.15.1 installed. I could unlock the archive there without any problems. I then upgraded to v2.16.1 and got the same problem as I was having on Firefox, except with a different UUID:

<img width="505" alt="Chrome" src="https://user-images.githubusercontent.com/4579020/86316570-6149d400-bc2d-11ea-8092-77188fb49d7c.png">

I tried downgrading by downloading the v2.15.1 release, running `npm install` and then `npm run release`, but the latter step exited immediately without apparently doing anything. I also tried `npm run dev:firefox` with the same results. Is there some other way to downgrade?

**Comment by perry-mitchell** [comment]
Hi @wodin - I'm sorry to hear you're having issues. What type of vault is it? Is it pure WebDAV, or is it an ownCloud/Nextcloud connection?

If the issue continues I'd advise removing the connection and re-creating it.

> Is there some other way to downgrade?

There isn't, as browsers don't have support for handling multiple builds in production.

**Comment by wodin** [comment]
Thanks for the response.

> What type of vault is it? Is it pure WebDAV, or is it an ownCloud/Nextcloud connection?

It's Apache with `mod_dav_fs`, i.e. "pure WebDAV", I suppose.

> If the issue continues I'd advise removing the connection and re-creating it.

I am convinced that the problem was caused by the upgrade, because it broke the first time I tried it in Firefox after the upgrade. Then I tried the older version in Chrome and that worked. Then manually upgraded it and it immediately broke.

However, surprisingly, removing the connection and re-creating it fixed it! :)

Thanks for the help.
